src/data/reducers/productDataReducer.jsx

```diff
@@ -1,16 +1,11 @@
-import { GET_PRODUCT_DATA, GET_PRODUCT_DATA_SUCCESS } from '../types/types';
+import { GET_PRODUCT_DATA_SUCCESS } from '../types/types';
 
 const initialState = {
   productData: {},
 };
 
 const productDataReducer = (state = initialState, action) => {
   switch (action.type) {
-    case GET_PRODUCT_DATA:
-      return {
-        ...state,
-      };
-
```

Comment:
Doesn't need to be in the reducer since the getProductData is simply calling on the API to get the productData for a specific product_id. Then once the data is available the GET_PRODUCT_DATA_SUCCESS action and reducer are called